source/layout.erb

```diff
@@ -31,19 +31,17 @@
       <%= partial "sidebar" %>
 
       <article>
-        <div class="chapter">
-          <h1>
-            <%= page_title %>
-            <a href="https://github.com/emberjs/guides/edit/master/source/<%= current_page.path.gsub('html', 'md') %>"
-               target="_blank" class="edit-page icon-pencil">Edit Page</a>
-          </h1>
-          <hr>
+        <h1>
+          <%= page_title %>
+          <a href="https://github.com/emberjs/guides/edit/master/source/<%= current_page.path.gsub('html', 'md') %>"
```

Comment:
Why is the pencil inside the H1? That doesn't make semantic sense to me. (I realize it was like that before your change.) Can you try moving it out and see if it would take much CSS readjustment?